isacc_messaging/api/isacc_record_creator.py

```diff
@@ -55,13 +55,15 @@ def __create_communication_from_request(self, cr):
 
             "payload": [p.as_json() for p in cr.payload],
             "sent": datetime.now().astimezone().isoformat(),
+            "sender": cr.sender,
             "recipient": [r.as_json() for r in cr.recipient],
             "medium": [{
                 "coding": [{
                     "system": "http://terminology.hl7.org/ValueSet/v3-ParticipationMode",
                     "code": "SMSWRIT"
                 }]
             }],
+            "note":[n.as_json() for n in cr.note] if cr.note else "",
```

Comment:
Will this set `"note": ""` on a Communication, even if it isn't set on the original CommunicationRequest?

Reply:
@ivan-c thank you for flagging that.  I wasn't sure what the correct value to set here.  I have now set it to `None` if `note` wasn't set on the original CommunicationRequest.  Let me know if you think that will be an issue though.